src/client/src/components/pages/participants/participantDetails/index.jsx

```diff
@@ -55,16 +66,20 @@ class index extends Component {
   render() {
 
     return (
-      <div className="detail-participant">
+      <>
```

Comment:
Cool, I didn't know about `<>`!

Reply:
Yeah :fire: 
Its a react feature that we can use it instead use

> `<React.Fragment> </React.Fragment>`
or
> `<Fragment> </Fragment>`

Reply:
👍 I have to tell my team!